proto/iotics/api/meta.proto

```diff
@@ -168,3 +143,58 @@ message SparqlUpdateResponse {
   // SPARQL update response headers
   Headers headers = 1;
 }
+
+// FKGQueryRequest describes a SPARQL query.
+message FKGQueryRequest {
+
+  // SPARQL query request payload.
+  message Payload {
+
+    // The desired result content type. Note that choosing an invalid result type for the type of query will result in
+    // an error status reported in the response. (See SparqlResultType for valid content-query type combinations.)
+    SparqlResultType resultContentType = 1;
+    // A UTF8-encoded SPARQL 1.1 query
+    bytes query = 2;
+  }
+
+  // SPARQL query request headers
+  Headers headers = 1;
+
+  // SPARQL query request scope
+  Scope scope = 2;
+
+  // SPARQL query request payload
+  Payload payload = 3;
+}
+
+// FKGQueryResponse is a part of a result for a SPARQL query request. Multiple chunks form a complete result. Related
+// chunks can be identified by a combination of:
+// - Client reference (in headers, set by caller)
+// - Chunk sequence number
+message FKGQueryResponse {
+
+  // Payload of the query result chunk
+  message Payload {
+
+    // Position of a chunk in result for a given request. The first chunk has a sequence number of 0.
+    uint64 seqNum = 1;
+
+    // Indicates whether this is the last chunk, for a specific request. Results for different requests can be
+    // identified by setting a unique clientRef in the request headers.
+    bool last = 2;
+
+    // Content type of the result.
+    SparqlResultType contentType = 3;
+
+    // Query result chunk, encoded according to actualType.
+    // Note that:
+    // - The maximum size of each chunk is host-specific.
```

Comment:
how does one find what the value is?

Reply:
There is no way yet but we could (later) expose a host high-level conf page. It could be part of an admin API, etc. For now it is visible via the AWS console

Reply:
OK - so it should be possible to document the current value here in the API?

Reply:
We could hint at a default but I think "hard coding" it here in the docs means we cannot change it.